README.md

```diff
@@ -664,49 +664,32 @@ print(type(post.content))
 # OUTPUT: Article
 # Article is very inclusive and all fields are optional, allowing any dict to become valid
 ```
-**Not Terrible Solutions:**
-1. Order field types properly: from the most strict ones to loose ones.
```

Comment:
oh, I didn't see that this point was deleted. I think it's still a relevant solution in cases when we cannot make a config for `Extra.forbid`, e.g. handling `ValueError` is too complicated in the context of an application.

Ordering field types properly is a stupid, yet working solution that requires almost zero changes in the code, so I think it should stay as a practice.

Reply:
It just fills unsafe, because when you change something in the pydantic class and change relational strictness, it'll affect all child classes/places where classes ordering by strictness - and some unexpected behavior could happen, because not it'll match to another class